parsely/parselyandroid/ParselyMetadata.java

```diff
@@ -5,51 +5,66 @@
 import java.util.HashMap;
 import java.util.Map;
 
-/*! \brief Represents article/video metadata to be passed to Parsely tracking.
+/*! \brief Represents article metadata to be passed to Parsely tracking.
  *
- *  This class is used to attach a metadata block to a Parse.ly pageview or video
- *  request. For most use cases this is only required for video data. Pageviews typically
- *  correspond to a URL which we can crawl on the customer site.
+ *  This class is used to attach a metadata block to a Parse.ly pageview
+ *  request. Pageview metadata is only required for URLs not accessible over the
+ *  internet (i.e. app-only content) or if the customer is using an "in-pixel" integration.
+ *  Otherwise, metadata will be gathered by Parse.ly's crawling infrastructure.
  */
 public class ParselyMetadata {
     public ArrayList<String> authors, tags;
-    public String canonical_url, section, thumbUrl, title;
+    public String canonicalUrl, section, thumbUrl, title;
     public Calendar pubDate;
 
+    /* \brief Create a new ParselyMetadata object.
+     *
+     * @param authors         List of authors for the article.
```

Comment:
We use the terminology "post or page" in our [metadata documentation](https://www.parse.ly/help/integration/jsonld/). I think "article" isn't the most futureproof.

Reply:
:+1: